pom.xml

```diff
@@ -374,22 +353,55 @@
     		<artifactId>org.apache.sling.event.api</artifactId>
     		<version>1.0.0</version>
     	</dependency>
+
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.serviceusermapper</artifactId>
+            <version>1.4.6</version>
+            <scope>provided</scope>
+        </dependency>
+
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.commons.testing</artifactId>
             <version>2.0.10</version>
         </dependency>
         <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.9.5</version>
-            <scope>test</scope>
+          <groupId>org.apache.sling</groupId>
```

Comment:
indentation is a bit off, we use 4 spaces in this pom.xml

Reply:
sure, now done.